.lsan.supp

```diff
@@ -7,3 +7,8 @@ leak:ld-linux-x86-64
 # OCCT
 leak:TKernel
 leak:TKXSBase
+
+# Potential mesa/VTK leak with incomplete callstack
+# forces us to hide all leaks from the libf3dSDKTests
+# https://gitlab.kitware.com/vtk/vtk/-/issues/18504
+leak:libf3dSDKTests
```

Comment:
@Meakk 

This is the big hammer solution to the problem. Alternatively I could
- Add a cmake flags to enable the SDK_ONLY_RENDER_WINDOW_TESTS
- Force the creation of an interactor in all our tests (basically, just add `getInteractor()` before the first render.

Wdyt ?

Reply:
I think filtering this is fine for now. Let's investigate later